software/tinyclr/tutorials/networking/spwf04sx-wifi.md

```diff
@@ -0,0 +1,446 @@
+# SPWF04Sx Wi-Fi Module
+---
+First introduced over twenty years ago, Wi-Fi has become the most popular wireless networking technology. Originally used to network desktop computers, Wi-Fi is being incorporated into an ever increasing number of electronic devices and is a driving force behind the Internet of things revolution.
```

Comment:
Make this intro more about this specific module. Link to the ST page for it, the datasheet, and TCP manual.